### PR TITLE
Merge changes of GS from dev to 3.0

### PR DESCRIPTION
**Pull request by FlorianJacta**
Improve Getting started links and imports

### CODE REVIEW

**Review by FabienLelaquais — APPROVED** [review]
👌

**Review by jrobinAV — APPROVED** [review]
Oh yeah!

File: docs/getting_started/index.md [hunk 1]
```diff
@@ -109,15 +108,19 @@ The graph involves:
     Hello Taipy!
     ```
 
+    [Get the scenario code](./hello_world_scenario.py){: .tp-btn target='blank'}
```
Comment (FabienLelaquais) [inline docs/getting_started/index.md]:
```suggestion
    [Get the code](./hello_world_scenario.py){: .tp-btn target='blank'}
```

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**